### PR TITLE
Align mobile nav items

### LINKED ISSUE #872: Line up mobile nav items
Icons on the bottom 3 should be horizontally inline with everything above. Right now kind of looks like those bottom 3 items are sub-items of 'Resources' which they are not. 

image

Label: good first issue

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

Closes #872 

## Screenshots or screen recordings

image

## Acceptance checklist

- [ x ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ x ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/components/Navbar/Mobile/MobileCollapse.tsx [hunk 1]
```diff
@@ -86,15 +86,17 @@ export default function MobileCollapse() {
...
-            <Menu.Item key="language-selector">
-              <NavLanguageSelector mobile />
-            </Menu.Item>
-            <Menu.Item key="theme-picker">
-              <ThemePickerMobile />
-            </Menu.Item>
-            <Menu.Item key="feedback">
-              <FeedbackFormButton mobile />
-            </Menu.Item>
+            <div className="top-left-nav" style={{ marginLeft: 15 }}>
```
Comment (johnnyd-eth) [inline src/components/Navbar/Mobile/MobileCollapse.tsx:89]: **Question: ** any reason we need the className here? I think I remember removing the CSS for this one, don't think it should be in any JSX

**Review by johnnyd-eth — CHANGES_REQUESTED** [review]
Just get rid of the 'top-left-nav' class then we'll be good to go 🚀 Also, I checked code and saw I had left another one somewhere else. Reckon you could remove it too?

**Comment by jmilldotdev** [comment]
Done

**Comment by johnnyd-eth** [comment]
Legend LGTM

**Review by johnnyd-eth — APPROVED** [review]

**Pull request merged**